modules/clients/src/test/java/org/apache/ignite/internal/processors/rest/ChangeStateCommandHandlerTest.java

```diff
@@ -87,23 +89,23 @@
     public void testActivateDeActivate() throws GridClientException {
         GridClientClusterState state = client.state();
 
-        boolean active = state.active();
+        boolean active = state.state() == ACTIVE;
 
         assertTrue(active);
 
-        state.active(false);
+        state.state(INACTIVE, true);
 
         IgniteEx ig1 = grid(0);
         IgniteEx ig2 = grid(1);
 
         assertFalse(ig1.active());
         assertFalse(ig2.active());
-        assertFalse(state.active());
+        assertEquals(state.state(), INACTIVE);
 
-        state.active(true);
+        state.state(ACTIVE, true);
```

Comment:
There is no cluster deactivation, `forceDeactivation` should be false.

Reply:
> There is no cluster deactivation, `forceDeactivation` should be false.

Previous invocations were considered forced regardless of certain state. Shouldn't we do the same way?

Reply:
Previous deprecated call was forced internally for both cases. I suggest do not forcing new public method call to cluster activation.